webcompat_migration_backup_db.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
+import datetime
+import time
+from subprocess import call
+
+import os
+from git import Repo, GitCommandError
+
+path = os.path.dirname(os.path.realpath(__file__))
+
+
+def run_db_migration():
+    current_ts = time.time()
+    time_stamp = datetime.datetime.fromtimestamp(current_ts).strftime('%Y-%m-%d-%H:%M:%S')
+    issue_backup_db = 'issues_backup_' + time_stamp + '.db'
+    repo = Repo(path)
+    assert not repo.bare
+    assert repo.refs['origin/master'] == repo.remotes.origin.refs.master
+    origin = repo.remotes.origin
+    index = repo.index
+    db_path = path + '/backup_db'
+    if os.path.exists(path + '/issues.db'):
+        if os.path.exists(db_path) and os.path.isdir(db_path):
+            #           check if a previous version of a backup exist and delete it
+            print 'Backup database already exists. Removing older version of the back up'
+            call('rm -r backup_db', shell=True)
```

Comment:
You are already importing the `os` module, so it seems not necessary here to import the subprocess module. You could directly use the [`os.remove`](https://docs.python.org/2/library/os.html#os.remove) for a file or [`shutil.rmtree`](https://docs.python.org/2/library/shutil.html#shutil.rmtree) for non-empty directory and its content. (there is `os.rmdir` for empty directories.)